src/serlo-editor/plugin/object.ts

```diff
@@ -16,17 +16,16 @@ import {
  * @param getFocusableChildren - Allows to override the default order of focusable children
  */
 export function object<Ds extends Record<string, StateType>>(
-  types: Ds,
-  getFocusableChildren: (children: { [K in keyof Ds]: { id: string }[] }) => {
-    id: string
-  }[] = (children) => {
-    return R.flatten<readonly FocusableChild[][]>(R.values(children))
-  }
+  types: Ds
 ): ObjectStateType<Ds> {
   type S = StateTypesSerializedType<Ds>
   type T = StateTypesValueType<Ds>
   type U = StateTypesReturnType<Ds>
 
+  const getObjectChildIds = (children: {
+    [K in keyof Ds]: { id: string }[]
+  }) => R.flatten<readonly FocusableChild[][]>(R.values(children))
```

Comment:
Does the naming of `FocusableChild` still make sense?

Reply:
Any proposals? 😄